source/form.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/icon/favicon.png">
+    <link rel="stylesheet" href="css/">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+  <body>
+    <h1 class="visually hidden" lang="en">CAT ENERGY</h1>
+    <header class="header">
+      <a class="header__logo logo">
+        <img class="logo__img" width="33" height="37" alt="Наш маскот." src="img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип «Кэт Энерджи»." src="img/icon/logo_main.svg">
+      </a>
+      <button class="header__toggle" type="button" aria-expanded="false" aria-controls="main-nav">
+        Открыть меню
+      </button>
+      <nav class="header__popover menu" id="main-nav">
+        <ul class="menu__content-list">
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="index.html">Главная</a>
+           </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="form">
+        <h2 class="form__title">Подбор программы</h2>
+        <p class="form__text">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+        <form class="form__filter filter" action="https://echo.htmlacademy.ru/" method="get">
```

Comment:
Не вижу смысла в get
```suggestion
        <form class="form__filter filter" action="https://echo.htmlacademy.ru/" method="POST">
```